TUM Campus App/PlanDetailsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PlanDetailsViewController.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 16.12.16.
+//  Copyright © 2016 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class PlanDetailsViewController: UIViewController, UIScrollViewDelegate {
+
+    @IBOutlet weak var scrollView: UIScrollView!
+    var imageView: UIImageView!
```

Comment:
Is there any way for you to get rid of these extra properties? It doesn't seem very clean to keep a reference to these views